plugins/zip-extractor/src/index.ts

```diff
@@ -28,8 +28,9 @@ export const ZipExtractor = () => {
             progress: 50,
             info: 'Uploading files',
           })
-          zipEntries.forEach(async (zipEntry) => {
+          const uploadPromises = zipEntries.map(async (zipEntry) => {
             if (
+              !zipEntry.entryName.startsWith('__MACOSX') &&
               !zipEntry.entryName.startsWith('.') &&
```

Comment:
Seems like this already should have solved the issue that I reported - the extraneous files all started with `.`. Why does the change in this PR fix the issue?

Reply:
@alexhollenbeck apparently `entryName` contains the entire path.